libs/interpret-text/src/lib/TextExplanationDashboard/Control/TextHightlighting.tsx

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { NeutralColors, SharedColors } from "@fluentui/theme";
+import React from "react";
+
+import { Utils } from "../CommonUtils";
+import { IChartProps } from "../Interfaces/IChartProps";
+
+const highlighted = {
```

Comment:
style needs to be moved to separated style file with office fabric mergeStyles function.

Reply:
oh good point, let me put it into a styles file